database/insertData/03_template_D_basicReviewTesting.js

```diff
@@ -359,14 +359,30 @@ exports.queries = [
                 }
               }
               {
-                actionCode: "trimResponses"
+                actionCode: "changeStatus"
                 trigger: ON_REVIEW_SUBMIT
                 sequence: 1
                 parameterQueries: {
                   reviewId: {
                     operator: "objectProperties"
                     children: ["applicationData.record_id"]
                   }
+                  newStatus: "Submitted"
+                }
+              }
+              {
+                actionCode: "trimResponses"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 2
+                parameterQueries: {
+                  reviewId: {
+                    operator: "objectProperties"
+                    children: ["applicationData.record_id"]
+                  }
+                  timestamp: {
+                    operator: "objectProperties"
+                    children: ["output.reviewStatusHistoryTimestamp"]
```

Comment:
ideally we would also do this for ON_APPLICATION_SUBMIT

Reply:
ef4d5dc0